src/__tests__/index.ts

```diff
@@ -95,23 +95,39 @@ beforeEach(() => {
 })
 
 describe('createVerifiableCredential', () => {
+  const issuer = ethrDidIssuer
```

Comment:
Is there any sense behind that reassigning / renaming?, otherwise I'd just stick to "ethrDidIssuer"